websites/P/Perplexity/presence.ts

```diff
@@ -0,0 +1,86 @@
+const presence = new Presence({
+		clientId: "893871116945350726",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	threadExportRegex = /([a-f\d]{8}(-[a-f\d]{4}){3}-[a-f\d]{12})/i;
+let recentSearchQuery: string = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "lg",
+			startTimestamp: browsingTimestamp,
+		},
+		privacy = await presence.getSetting<boolean>("privacy"),
+		hideButtons = await presence.getSetting<boolean>("hideButtons"),
+		currentSearch = new URL(document.location.href).searchParams.get("q"),
+		pageInput = document.querySelector<HTMLInputElement>(".md\\:mb-md"),
+		currentThread = document.location.pathname.match(threadExportRegex),
+		currentPage = document.location.pathname.split("/")[1];
+
+	if (currentSearch && currentSearch !== recentSearchQuery)
+		recentSearchQuery = currentSearch;
+
+	presenceData.details = "Home";
+
+	if (!currentPage) return presence.setActivity(presenceData);
+
+	switch (currentPage.toLowerCase()) {
+		case "search":
+			presenceData.details = "Searching for";
+			presenceData.smallImageKey = "search";
```

Comment:
We use imgur links for all non element large & small image keys
```suggestion
			presenceData.smallImageKey = "https://i.imgur.com/wYVlwJX.png";
```